src/mpi/romio/adio/ad_gpfs/ad_gpfs_aggrs.c

```diff
@@ -462,7 +462,7 @@ void ADIOI_GPFS_Calc_my_req(ADIO_File fd, ADIO_Offset * offset_list, ADIO_Offset
 #ifdef AGGREGATION_PROFILE
     MPE_Log_event(5024, 0, NULL);
 #endif
-    *count_my_req_per_proc_ptr = ADIOI_Calloc(nprocs, sizeof(int));
+    *count_my_req_per_proc_ptr = ADIOI_Calloc(nprocs, sizeof(**count_my_req_per_proc_ptr));
```

Comment:
Not sure the double dereference is pleasent on the eye, but I'll let it slide ;)